packages/expo-dev-launcher/ios/Tests/EXDevLauncherControllerTest.swift

```diff
@@ -18,17 +18,24 @@ class EXDevLauncherControllerTest: QuickSpec {
       expect(sharedInstance).to(be(EXDevLauncherController.sharedInstance()))
     }
 
-    // TODO: fix
-    xit("extraModulesForBridge should return essential modules") {
+    it("extraModulesForBridge should return essential modules") {
       let module = EXDevLauncherController.sharedInstance()
 
       let modules = module.extraModules(for: nil)!
 
-      expect(modules.count).to(equal(4))
+      expect(modules.count).to(equal(10))
```

Comment:
updated this to include all the vendored modules that are now also returned, which I think is correct since they would be considered "essential" for the dev launcher app to run properly -- @ajsmth does that sound correct to you?

Reply:
yes thank you!